tiled/client/base.py

```diff
@@ -866,11 +866,27 @@ def metadata_revisions(self):
 
         return self._metadata_revisions
 
-    def delete_tree(self):
-        endpoint = self.uri.replace("/metadata/", "/nodes/", 1)
+    def delete(self, recursive: bool = False, external_only: bool = True) -> None:
+        """Delete the node and it's contents, if any.
+
+        Parameters
+        ----------
+        recursive : bool, optional
+            If True, descend into sub-nodes and delete their contents too.
+            Defaults to False.
+        external_only : bool, optional
+            If True, only delete externally-managed data keys. Defaults to True.
```

Comment:
```suggestion
            If True, only delete externally-managed data. Defaults to True.
```

Avoiding "data keys" as an overloaded term (bluesky). And the sentence works fine with just "data".